src/status_im/config.cljs

```diff
@@ -167,5 +167,5 @@
 
 (def default-kdf-iterations 3200)
 
-(def community-accounts-selection-enabled? (enabled? (get-config :ACCOUNT_SELECTION_ENABLED "0")))
+(def community-accounts-selection-enabled? true)
```

Comment:
can we move this to feature flags now that I merged the UI for feature flags pr? i.e https://github.com/status-im/status-mobile/pull/18602

 (not saying it has to be done in this pr)
cc @ilmotta @ajayesivan @FFFra

Reply:
I believe @ilmotta has done some work related to this. @ilmotta, if you don't have any objections, I can address this in a follow-up PR.

@Parveshdhull, for this PR, I suggest reverting these changes.